third_party/terraform/resources/resource_sql_database_instance.go.erb

```diff
@@ -403,42 +436,63 @@ func resourceSqlDatabaseInstance() *schema.Resource {
 						"ca_certificate": {
 							Type:         schema.TypeString,
 							Optional:     true,
+							<% unless version == 'ga' -%>
```

Comment:
I'm curious why these are different between beta and GA

Reply:
Removed